priv/repo/migrations/20240502105156_migrate_suma_settings_sti.exs

```diff
@@ -0,0 +1,15 @@
+defmodule Trento.Repo.Migrations.MigrateSumaSettingsSti do
+  use Ecto.Migration
+
+  def change do
+    alter table(:settings) do
+      add :suse_manager_settings_url, :string
+      add :suse_manager_settings_username, :string
+      add :suse_manager_settings_password, :binary
+      add :suse_manager_settings_ca_cert, :binary
+      add :suse_manager_settings_ca_uploaded_at, :utc_datetime_usec
+    end
+
+    drop table(:software_update_settings)
```

Comment:
I was about to say that we maybe need to migrate the content from `software_update_settings` to `settings`, but since we never released this yet, I guess this should be just fine.